packages/legend-query-builder/src/__lib__/QueryBuilderTelemetryHelper.ts

```diff
@@ -24,10 +24,12 @@ import {
 
 type QueryExecution_TelemetryData = GraphManagerOperationReport & {
   dependenciesCount: number;
+  state?: object | undefined;
```

Comment:
Don't like this type being an object here. Is this suppose to be the querySource or querySDLC info here ? 
Can we create a type here that makes sense. or would we reach some circular dependencies?